cmd/argo/commands/submit.go

```diff
@@ -30,20 +35,25 @@ func NewSubmitCommand() *cobra.Command {
 		submitOpts    util.SubmitOpts
 		cliSubmitOpts cliSubmitOpts
 		priority      int32
+		from          string
 	)
 	var command = &cobra.Command{
```

Comment:
I think this should change. We should not make CLI commands have multiple responsibilities like this. 

Please can you implement something like `argo cron run...` instead?

Reply:
See https://en.wikipedia.org/wiki/Single_responsibility_principle

Reply:
As mentioned [above](https://github.com/argoproj/argo/pull/2003#discussion_r376520027) and discussed in #1963, @jessesuen and I decided that we should follow the `kubectl` pattern and implement this as is.

Reply:
@jessesuen please can you feedback on this?

Reply:
actually - I'll do a u-turn on this - I like it as it as you've implemented it

does it work for, say?

`argo submit my-my --from cronwf/my-cron-wf`